pokete_classes/movemap.py

```diff
@@ -66,8 +66,7 @@ def text(self, x, y, inp_arr):
         self.assure_distance(x, y, 17, 10)
         self.multitext.rechar("")
         self.multitext.add(self, x - self.x + 1, y - self.y)
-        arr = [i + (" >" if j != len(inp_arr) - 1 else "")
-                    for j, i in enumerate(inp_arr)]
+        arr = ["< " + str + " >" for str in inp_arr]
```

Comment:
```suggestion
        arr = [" < " + i + (" >" if j != len(inp_arr) - 1 else "")
               for j, i in enumerate(inp_arr)] 
```

Like I mentioned before the closing bracket just indicates that text will follow and I think I want to keep it like that.

Also there is a space missing in before of the opening bracket.

Also calling the variable `str` is not a good idea, because there is a python buildin type called `str` which would get overwritten.